app/controllers/posts/comments_controller.rb

```diff
@@ -8,6 +8,7 @@ def new
   def create
     @comment = @post.comments.build(comment_params.merge(user_id: current_user.id))
     if @comment.save
+      UserMailer.with(comment: @comment).commented.deliver_now if @post.user.email.present? && @post.user_id != @comment.user_id
```

Comment:
userをパラメータにしたほうがシンプルになりそうだなとおもった。
```suggestion
      UserMailer.with(user: @post.user).commented.deliver_now if @post.user.email.present? && @post.user_id != @comment.user_id
```

Reply:
userだけでしょうか？？
commentのpostの情報がない状態だと、postへのリンクを生成できないと思うのですが、どうでしょう？？

Reply:
skipで良さそう

Reply:
アンカーリンクつくるため、commentの情報が必要なのでSKIPしてOK